scene/gui/color_picker.h

```diff
@@ -64,6 +71,15 @@ class ColorPicker : public BoxContainer {
 	GDCLASS(ColorPicker, BoxContainer);
 
 public:
+	enum ColorModeType {
```

Comment:
```suggestion
	enum ColorMode {
```
Picker shape could be changed too, but can be left for later.

Reply:
Can't be of same name as ColorMode class
<img width="503" alt="image" src="https://user-images.githubusercontent.com/79760854/175773394-2839bb2f-b566-43cd-9faf-8fb088bd5bdf.png">

Reply:
Ok, makes sense.